### PR TITLE
Bump runtime to 23.08

### LINKED ISSUE #7: runtime org.freedesktop.Platform branch 20.08 is end-of-life
org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.

### PR DESCRIPTION
**Pull request by bbhtt**
Closes https://github.com/flathub/org.quetoo.Quetoo/issues/7

### CODE REVIEW

**Comment by bbhtt** [comment]
Runs

**Comment by AsciiWolf** [comment]
@jdolan Any chance this could get merged?

**Comment by AsciiWolf** [comment]
bot, build

**Comment by AsciiWolf** [comment]
Needs AppStream metadata update.

**Comment by bbhtt** [comment]
> @jdolan Any chance this could get merged?

If you can test I'll merge. I doubt the maintainer is active anymore here.

**Comment by AsciiWolf** [comment]
Replaced by #10.

File: org.quetoo.Quetoo.appdata.xml [hunk 1]
```diff
@@ -14,35 +18,21 @@
...
+  <launchable type="desktop-id">org.quetoo.Quetoo.desktop</launchable>
   <screenshots>
-    <screenshot type="default">https://farm3.staticflickr.com/2818/33459192014_2e76a22a09_c.jpg</screenshot>
-    <screenshot>https://farm8.staticflickr.com/7191/6840396554_70726caf45_c.jpg</screenshot>
-    <screenshot>https://farm8.staticflickr.com/7044/6840397000_34e7699f1f_c.jpg</screenshot>
+    <screenshot type="default">
+      <image>https://farm3.staticflickr.com/2818/33459192014_2e76a22a09_c.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image>https://farm3.staticflickr.com/2818/33459192014_2e76a22a09_c.jpg</image>
```
Comment (AsciiWolf) [inline org.quetoo.Quetoo.appdata.xml]:
```suggestion
      <image>https://farm8.staticflickr.com/7191/6840396554_70726caf45_c.jpg</image>
```

**Review by AsciiWolf — CHANGES_REQUESTED** [review]
Duplicated screenshot fix.

**Pull request merged**